examples/nextjs/starter/src/app/sitemap.ts

```diff
@@ -0,0 +1,17 @@
+import type { MetadataRoute } from 'next';
+import {
+	generateSitemaps as generateWPSitemaps,
```

Comment:
`generateSitemaps` does not look like a good name for this since it gets an array of objects with `id` and not a sitemap(s). I suggest `getSitemapLocations` or something along those lines.

Reply:
Not resolved